app/models/patient.rb

```diff
@@ -368,6 +371,47 @@ def latest_transfer
     end
   }
 
+  # Gets the current date in the patient's timezone
+  def curr_date_in_timezone
+    Time.now.getlocal(address_timezone_offset)
```

Comment:
Excellent; Can you please add Palau while you're doing this? https://github.com/SaraAlert/SaraAlert/blob/master/app/helpers/patient_helper.rb#L254

Reply:
Actually Palau is already there which is why I didn't have to make the change.